packages/features/embed/Embed.tsx

```diff
@@ -578,6 +584,19 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
   const defaultConfig = {
     layout: BookerLayouts.MONTH_VIEW,
   };
+
+  const paletteDefaultValue = (paletteName: string) => {
```

Comment:
@asadath1395 Removed the usage of `useCallback` which makes sense if we are passing functions to components causing it to change reference. Here we are using the function call result, so that isn't needed.